earthly-entrypoint.sh

```diff
@@ -9,7 +9,13 @@ if [ ! -f "$earthly_config" ]; then
 
   # Apply global configuration
   if [ -n "$GLOBAL_CONFIG" ]; then
-    earthly --config $earthly_config config global "$GLOBAL_CONFIG"
+    if [ "$( echo "$GLOBAL_CONFIG" | wc -l)" -gt 1 ]; then
+      # TODO FIXME once this issue is fixed, we can revert this hack
```

Comment:
probably best to just fix https://github.com/earthly/earthly/issues/1764 first rather than commit this work-around.

this was here just to show that it can work once this issue is fixed.